src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -227,6 +227,22 @@
   private static final String COMPACT_MAP_ENTRIES_DISPLAY = "Compact Map Entries";
   private static final boolean COMPACT_MAP_ENTRIES_DEFAULT = true;
 
+  public static final String DATA_STREAM_TIMESTAMP_CONFIG = "data.stream.timestamp";
+  private static final String DATA_STREAM_TIMESTAMP_DOC = String.format(
+      "The timestamp to use for the @timestamp field in messages sent to data stream. "
```

Comment:
Let's add a brief explanation that ES data streams require documents to contain this field
```suggestion
     <explanation here>  "The Kafka record field to use as the timestamp for the ``@timestamp`` field in documents sent to a data stream. "
```